app/code/Magento/MediaStorage/Model/File/Validator/NotProtectedExtension.php

```diff
@@ -93,7 +96,7 @@ protected function _initProtectedFileExtensions()
      * Return list with protected file extensions
      *
      * @param \Magento\Store\Model\Store|string|int $store
-     * @return string|string[]
```

Comment:
This function will NEVER return an array of strings. The only function used in this will return mixed.

Reply:
Changed to "string|null"